<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userspace</class>
 <widget class="QWidget" name="userspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人</string>
  </property>
  <widget class="QTableWidget" name="tableTickets">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>801</width>
     <height>341</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <column>
    <property name="text">
     <string>航班号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>座位等级</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出发地</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>目的地</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ReturnButtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#ReturnButtn{
	image: url(:/Image/Imageres/back.png);
    background: transparent; /* 背景透明 */
    border: none; /* 移除边框 */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSet1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSet2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccount">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>811</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
background-image: url(:/Imageres/airBg.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>tableTickets</zorder>
  <zorder>ReturnButtn</zorder>
  <zorder>labelSet1</zorder>
  <zorder>labelSet2</zorder>
  <zorder>labelName</zorder>
  <zorder>labelAccount</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
